test/after_commit_test.rb

```diff
@@ -119,6 +136,8 @@ def teardown
     end
   end
 
+  def test_after
+
```

Comment:
!?

Reply:
woops. left over and will be fixed :D but since I can't run the tests at all…